.github/pull_request_template.md

```diff
@@ -1,52 +1,58 @@
 ### Description
 
-<!-- A detailed explanation of the changes in your PR. Feel free to remove this section if the title of your PR is sufficiently descriptive. -->
+<!--
+A detailed explanation of the changes in your PR. Feel free to remove this
+section if the title of your PR is sufficiently descriptive. To learn more
+about contributing to this project, check "*.md" files under:
+    https://github.com/stackrox/stackrox/tree/master/.github
+-->
 
 change me!
 
 ### User-facing documentation
 
-(*must be* 2 items and both *must be* checked)
-<!-- Remove conflicting items that won't be checked. -->
-
-- [ ] CHANGELOG is updated
-- [ ] CHANGELOG update is not needed
-- [ ] Documentation PR is created and linked above
-- [ ] Documentation is not needed
+- [ ] CHANGELOG is updated **OR** update is not needed
+- [ ] documentation PR is created and linked above **OR** is not needed
```

Comment:
Please link this https://spaces.redhat.com/display/StackRox/Submitting+a+User+Documentation+Pull+Request

Reply:
Good call, thanks!